.pre-commit-config.yaml

```diff
@@ -4,7 +4,9 @@ repos:
   hooks:
   - id: black
     language_version: python3.8
+    args: ["--line-length=80"]
```

Comment:
@misaacs i ran into the issue you mentioned about conflicting line lengths and added these arts to resolve the conflict